lib/smart_answer/calculators/uk_benefits_abroad_calculator.rb

```diff
@@ -30,5 +30,24 @@ def social_security_countries_bereavement_benefits?
       (COUNTRIES_OF_FORMER_YUGOSLAVIA +
       %w(barbados bermuda canada guernsey jersey israel jamaica mauritius new-zealand philippines turkey usa)).include?(country)
     end
+
+    def benefits?
+      benefits.present? && benefits.is_a?(Array) && valid_benefits?
```

Comment:
As per [this comment](https://github.com/alphagov/smart-answers/pull/3180#discussion_r132136772) can you please remove `benefits.present?` ? If this is a common pattern, it may even be nice to extract some sort of helper class?